department-of-veterans-affairs/caseflow
# TYLERB/APPEALS-59606: Queue Table Filter Get Params Error

Resolves [APPEALS-59606](https://jira.devops.va.gov/browse/APPEALS-59606)

# Description
Updated the queue table deep link method to fix an issue where it was only adding the last filter type (e.g. issue type, appeal type) to the get parameters when selecting a new filter.

## Acceptance Criteria
- [ ] Filter params are once again properly added to the get parameters 

## Testing Plan

1. Login as a vha user and navigate to the decision review queue
2. Select an issue type filter and verify it is added to the get params
![image](https://github.com/user-attachments/assets/5a448136-124d-43bc-8de1-7b29c122b755)

3. Select a decision review type filter and verify that the decision review type params are also added to the get params
![image](https://github.com/user-attachments/assets/6678a1bd-05ed-49b2-be99-29802ed47433)

4. Reload the page and verify that both filters are reapplied to the decision review queue from the get params
5. Remove the get params from the url and reload the page to verify that the filter preservation is still working for both filter types


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
